{{cookiecutter.repo_name}}/CHANGELOG.rst

```diff
@@ -1,8 +1,8 @@
 Change Log
-----------
+==========
```

Comment:
@jmbowman I don't have a screenshot, but with the formatting before edx/edx-enterprise#35 the changelog ToC looked like

    * Added
    * Added
    * Added
    * Added
    ...

As opposed to:

    * [0.21.0] - 2017-01-30
    * [0.20.0] - 2017-01-30
    ....

Having a closer look now I see what the problem was - at some moment "correct" schema of (`-`, `~`, `_`) was replaced by (`-`, `~`, `-`) in edx-enterprise by mistake. django-user-tasks uses another different (but working) schema of (`-`, `~`, `+`). [Sphinx docs](http://www.sphinx-doc.org/en/stable/rest.html#sections) suggest using yet another schema of (`-`, `^`, `"`).

Sphinx is flexible about which particular symbol to use for which nestedness level, but in order to be consistent I'd suggest we standardize on the format proposed in Sphinx docs, which in turn is adopted from [Python’s Style Guide for documenting ](https://docs.python.org/devguide/documenting.html#style-guide).

Regarding heading level - OK, I'll move it one level down; but edx-enterprise pypi looks roughly  the same (and IMHO definitely not worse than django-user-tasks) with changelog at the same header level: https://pypi.python.org/pypi/edx-enterprise/0.27.0